src/DotLiquid/StandardFilters.cs

```diff
@@ -433,14 +433,10 @@ public static IEnumerable SortNatural(object input, string property = null)
         private static IEnumerable SortInternal(StringComparer stringComparer, object input, string property = null)
         {
             List<object> ary;
-            if (input is IEnumerable<Hash> enumerableHash && !string.IsNullOrEmpty(property))
-                ary = enumerableHash.Cast<object>().ToList();
-            else if (input is IEnumerable enumerableInput)
-                ary = enumerableInput.Flatten().Cast<object>().ToList();
+            if (input is IEnumerable enumerable)
+                ary = enumerable.Cast<object>().ToList();

```

Comment:
I guess these are the lines that need to be reverted to fix @daviburg issue with flattening.